neo/Network/RPC/HttpService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Neo.IO.Json;
+using Neo.Network.RPC.Model;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Neo.Network.RPC
+{
+    public class HttpService : IDisposable
```

Comment:
I think we can remove this class, and move `HttpClient` into `RpcClient`.

Reply:
👌